### PR TITLE
Client - SEO improvement - add meta tag in page

### PR DESCRIPTION
**Pull request by MuhweziDeo**
Implement SEO for home page and detail page

### CODE REVIEW

File: geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html [hunk 1]
```diff
@@ -42,8 +42,9 @@
                 geonodeUrl: '{{ SITEURL }}',
                 geoserverUrl: '{{ OGC_SERVER.default.PUBLIC_LOCATION|default:"" }}',
                 printUrl: '{{ GEOSERVER_PUBLIC_LOCATION }}pdf/info.json',
-                bingApiKey: '{% bing_api_key %}'
-            }
+                bingApiKey: '{% bing_api_key %}',
+                siteName: '{{ SITE_HOST_NAME|default:"Geonode" }}',
```
Comment (luorlandini) [inline geonode_mapstore_client/templates/geonode-mapstore-client/_geonode_config.html:46]: could you please, change SITE_HOST_NAME in SITE_NAME

File: geonode_mapstore_client/client/js/components/home/Meta.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <title>{siteName}</title>
+        <meta name="description" content={content} />
+        <meta property="og:title" content={siteName + " " +  content} />
+        <meta property="og:site_name" content={siteName} />
+        <meta property="og:url" content={contentURL} />
+        <meta property="og:description" content={content} />
+        <meta property="og:image" content={logo} />
+
+        <meta name="twitter:site" content={siteName} />
+        <meta name="twitter:card" content={logo} />
```
Comment (luorlandini) [inline geonode_mapstore_client/client/js/components/home/Meta.jsx:15]: please, change in `<meta name="twitter:card" content="summary_large_image">`

**Comment by luorlandini** [comment]
@MuhweziDeo could you please, please move Meta component at a higher level, like Media component and add unit test,
thanks

To enable the metatag in the document viewer,  could you please, add the component also in mediaviewerplugin

**Comment by luorlandini** [comment]
@MuhweziDeo thanks for adding Metatag component in DocumentViewer route, could you please add it, also in GeoStoryViewer, LayerViewer, MapViewer routes. Thanks

**Comment by MuhweziDeo** [comment]
@luorlandini I have implemented the requests please review

**Review by luorlandini — APPROVED** [review]

**Pull request merged**